kitchen.recipes.yml

```diff
@@ -220,6 +220,50 @@ suites:
 #        - recipe:aws-parallelcluster-install::directories
 #        - recipe:aws-parallelcluster-install::python
 #        - recipe:aws-parallelcluster-install::cloudwatch_agent
+  - name: fs_update
+    run_list:
+      - recipe[aws-parallelcluster::add_dependencies]
```

Comment:
If I remember correctly, we have a recipe that sets up the envars for Chef.
Shouldn't that recipe be on top of every run_list?